keep-notes/src/components/NotesInput.jsx

```diff
@@ -1,33 +1,102 @@
-import React, { useState } from 'react'
-import "../css/notes-input.css"
-import {ChromePicker} from "react-color"
+import React, { useState } from "react";
+import { ChromePicker } from "react-color";
+import axios from "axios";
+import { useAuth } from "../context/auth-context";
+import "../css/notes-input.css";
 
-function NotesInput() {
 
+function NotesInput() {
+  
+  const {token} = useAuth();
   const [colorPalette, setColorPalette] = useState(false);
-  const [color , setColor] = useState("#FFFFFF");
+  const [color, setColor] = useState("#FFFFFF");
+
+  const date = new Date();
+
+  const initialNote = {
+    title: "",
+    content: "",
+    dateCreated: `${date.getDate()}/${
+      date.getMonth() + 1
+    }/${date.getFullYear()}`,
+    label: "",
+    backgroundColor: "#FFFFFF",
+  };
+
+  const [note, setNote] = useState(initialNote);
+
+  const notesHandler = async () => {
+    try{
+      const response = await axios.post("/api/notes",note,{headers:{authorization: token}})
+
+      if(response.status === 200 || response.status === 201){
+        console.log("check response", response.data.notes)
+
+      }
+    }
+    catch(err){
+      console.log(err.message);
+    }
+  }
 
   return (
     <>
-    <div className="notes-input-container">
-       <header className='notes-header'>
-           <input className='notes-title' type="text" placeholder='Title' />
-           <p><i className="bi bi-pin-fill"></i></p>
-       </header>
-      <section>
-          <textarea placeholder='Type Notes' className='notes-area'></textarea>
-      </section>
-      <footer className='notes-footer'>
+      <div className="notes-input-container">
+        <header className="notes-header">
+          <input
+            className="notes-title"
+            type="text"
+            placeholder="Title"
+            value={note.title}
+            onChange={(e) => setNote({ ...note, title: e.target.value })}
```

Comment:
Try to use updater function for state updates

Reply:
creating a new branch for adding notes will surely make changes there.